cmd/argot/main.go

```diff
@@ -0,0 +1,169 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/awslabs/ar-go-tools/analysis"
+	"github.com/awslabs/ar-go-tools/cmd/argot/backtrace"
+	"github.com/awslabs/ar-go-tools/cmd/argot/cli"
+	"github.com/awslabs/ar-go-tools/cmd/argot/compare"
+	"github.com/awslabs/ar-go-tools/cmd/argot/defers"
+	"github.com/awslabs/ar-go-tools/cmd/argot/dependencies"
+	"github.com/awslabs/ar-go-tools/cmd/argot/maypanic"
+	"github.com/awslabs/ar-go-tools/cmd/argot/packagescan"
+	"github.com/awslabs/ar-go-tools/cmd/argot/reachability"
+	"github.com/awslabs/ar-go-tools/cmd/argot/render"
+	"github.com/awslabs/ar-go-tools/cmd/argot/statistics"
+	"github.com/awslabs/ar-go-tools/cmd/argot/taint"
+	"github.com/awslabs/ar-go-tools/cmd/argot/tools"
+)
+
+const usage = `Argot: Automated Reasoning Go Tools
+Usage:
+  argot [tool] [options] <Go file path(s)>
+Tools:
+  - taint: performs a taint analysis on a given program
+  - backtrace: identifies backwards data-flow traces from function calls
+  - cli: terminal-like interface for parts of the analysis
+  - compare: prints a comparison of the functions that are reachable according to two different analyses, and the functions that appear in the binary
+  - dependencies: prints the dependencies of a given program
+  - maypanic: performs a may-panic analysis on a given program
+  - packagescan: scans imports in packages
+  - reachability: analyzes the program an prints the functions that are reachable within it
+  - render: renders a graph representation of the callgraph, or prints the program's SSA form
+  - ssa-statistics: prints statistics about the SSA representation of the program
+Examples:
+  Run the interactive CLI: argot cli --config=config.yaml main.go
+  Run the taint analysis: argot taint --config=config.yaml main.go`
+
+//gocyclo:ignore
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Fprintf(os.Stderr, "error: expected subcommand\n%s", usage)
```

Comment:
Missing a new line after the usage message: 
<img width="517" alt="Screenshot 2024-09-12 at 9 13 19 AM" src="https://github.com/user-attachments/assets/4e25defc-8876-4d67-a450-42abb25635e6">